src/SME.SGP.WebClient/src/paginas/Principal/principal.js

```diff
@@ -1,5 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
+import Card from '../../componentes/card';
+import Grid from '../../componentes/grid';
+import CardLink from '../../componentes/cardlink';
+import Row from '../../componentes/row';
+import Alert from '../../componentes/alert';
+import styled from 'styled-components';
 
-export default function Principal() {
-  return <React.Fragment></React.Fragment>;
+const Principal = (props) => {
+
+  const FREQUENCIA_TYPE = "frequencia";
+  const CICLOS_TYPE = "ciclos";
+  const ANUAL_TYPE = "anual";
+
+  const [escolaSelecionada, setEscolaSelecionada] = useState(false);
+
+  const ehDisabled = (tipo) => {
+
+    if (!escolaSelecionada)
+      return true;
+
+    console.log(tipo);
+    if (tipo === CICLOS_TYPE)
+      return !cicloLiberado();
+
+    return false;
+  }
+
+  const cicloLiberado = () => {
```

Comment:
É necessária uma function que só retorna false?

Reply:
sim, ela vai ser alterada quando o filtro for concluído. Preciso de algumas informações dele ai por enquanto eu deixei o esqueleto ja pronto, inclusive para o QA testar